base/pre-stop

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+supervisorctl stop all
+/etc/init.d/clearwater-etcd decommission
+
```

Comment:
blank line

Reply:
Also, i assume this probably needs to have the right permissions, but i can't see that here. I'm guessing all is good though as you've tested this live.